impeller/renderer/backend/vulkan/allocator_vk.h

```diff
@@ -31,6 +31,7 @@ class AllocatorVK final : public Allocator {
   std::weak_ptr<DeviceHolder> device_holder_;
   ISize max_texture_size_;
   bool is_valid_ = false;
+  bool supports_lazy_memory_ = false;
```

Comment:
I didn't put this in caps since its not clear that anything outside of the allocator needs to know yet...

Reply:
Perhaps call this `supports_memoryless_textures_` since we have used the term before and are familiar with it?